src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -725,44 +855,190 @@ function wait_for_dynamo_frontend()
+  }
+}
+EOF
+}
 
-  local genai_perf_arguments=$(array_to_args genai_perf_args)
-  log "Launching genai-perf with cmd: ${dynamo_args["genai-perf-cmd"]} $genai_perf_arguments ${genai_perf_args["--extra-args"]}"
+function setup_storage_cache_dir()
+{
+  # Use a global variable that can be exported
+  STORAGE_CACHE_DIR="$STORAGE_CACHE_DIR/${TEST_USER}/${dynamo_args["frontend-node"]}/${dynamo_args["connector"]}/cache"
+  rm -rf ${STORAGE_CACHE_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)